packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -147,8 +141,9 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
           [`:hover .${classNames.thumb}`]: slideBoxActiveThumbStyles,
           [`:active .${classNames.zeroTick}`]: slideBoxActiveZeroTickStyles,
           [`:hover .${classNames.zeroTick}`]: slideBoxActiveZeroTickStyles,
-          // eslint-disable-next-line deprecation/deprecation
-          ...getEdgeChromiumNoHighContrastAdjustSelector(),
+          [HighContrastSelector]: {
+            forcedColorAdjust: 'none',
```

Comment:
why not `...getHighContrastNoAdjustStyle(),` here?

Reply:
Because it changes the slider's HC styles. In IE, the top part of the slider is unfilled. In Edge/Chrome it is filled. I don't know the correct redlines and my goal was to not change any styling.